### PR TITLE
feat(embed): wording experiment iteration 2 - open and fork

### PR DESCRIPTION
**Pull request by tromika**
| A | B |
| - | - |
| Screenshot 2021-07-21 at 11 27 54 | image |

As per https://linear.app/codesandbox/issue/GRO-37/embed-cta-iteration-2-open-and-fork

Experiment to test out the Open and Fork wording for the CTA button in embeds

### CODE REVIEW

**Comment by lbogdan** [comment]
Build for latest commit https://github.com/codesandbox/codesandbox-client/pull/6049/commits/4893ede99fd95d7649ba1a2520f03f12c1e29ab0 is at https://pr6049.build.csb.dev/s/new.

File: packages/app/src/embed/components/Actions/index.js [hunk 1]
```diff
@@ -27,7 +26,7 @@ export function GlobalActions({
   previewVisible,
   initialPath,
 }) {
-  const experimentPromise = useExperimentResult('embed-open-wording');
+  const experimentPromise = useExperimentResult('embed-open-wording-iteration2');
```
Comment (danilowoz) [inline packages/app/src/embed/components/Actions/index.js:29]: Just make sure you created this experiment in both environments: production and staging.

**Review by danilowoz — APPROVED** [review]
Looks go to me 🚀

File: packages/app/src/embed/components/Actions/index.js (hunk 1 above)
Comment (tromika) [reply packages/app/src/embed/components/Actions/index.js:29]: Yeah, sure thing!

**Pull request merged**